tests/model/connection.test.js

```diff
@@ -17,264 +17,133 @@ var validPort3 = new Port('port-3', layer1, new Coord(0, 10));
 var validPort4 = new Port('port-4', layer2, new Coord(0, 15));
 var validPort5 = new Port('port-5', layer1, new Coord(0, 20));
 var validPort6 = new Port('port-6', layer2, new Coord(5, 0));
-var sourceComponent = new Component('source', 'unique-id-1', [layer1, layer2], 10, 20, 'entity', [validPort1, validPort2]);
-var sink1Component = new Component('sink1', 'unique-id-2', [layer1, layer2], 10, 20, 'entity', [validPort3, validPort4]);
-var sink2Component = new Component('sink2', 'unique-id-3', [layer1, layer2], 10, 20, 'entity', [validPort5, validPort6]);
+var sourceComponent = new Component('source', 'unique-id-1', 10, 20, 'entity', [validPort1, validPort2]);
+var sink1Component = new Component('sink1', 'unique-id-2', 10, 20, 'entity', [validPort3, validPort4]);
+var sink2Component = new Component('sink2', 'unique-id-3', 10, 20, 'entity', [validPort5, validPort6]);
```

Comment:
Since we check against these for validation, we should make sure that they are independent (specifically for the x/ySpan here)

Reply:
They have different values now.